Shoko.TrayService/App.xaml.cs

```diff
@@ -1,174 +1,167 @@
-﻿using System;
+﻿#region
+using System;
+using System.ComponentModel;
 using System.Diagnostics;
 using System.Drawing;
 using System.Linq;
-using System.Net.NetworkInformation;
+using System.Net;
 using System.Net.Sockets;
-using System.Runtime.InteropServices;
 using System.Windows;
 using System.Windows.Controls;
 using Hardcodet.Wpf.TaskbarNotification;
 using NLog;
-using Shoko.Models.Interfaces;
-using Shoko.Server;
-using Shoko.Server.Providers.AniDB;
-using Shoko.Server.Providers.AniDB.UDP.Generic;
-using Shoko.Server.Providers.AniDB.UDP.Info;
+using Shoko.Server.Commands;
 using Shoko.Server.Server;
 using Shoko.Server.Settings;
 using Shoko.Server.Utilities;
+#endregion
+namespace Shoko.TrayService;
 
-namespace Shoko.TrayService
+/// <summary>
+/// Interaction logic for App.xaml
+/// </summary>
+public partial class App
 {
-    /// <summary>
-    /// Interaction logic for App.xaml
-    /// </summary>
-    public partial class App : Application
-    {
-        public static TaskbarIcon Icon { get; set; }
-        private static readonly Logger Logger = LogManager.GetCurrentClassLogger();
-        private void OnStartup(object a, StartupEventArgs e)
-        {
-            Console.CancelKeyPress += (sender, args) => Dispatcher.Invoke(() =>
-            {
-                args.Cancel = true;
-                Shutdown();
-            });
-            Icon = new TaskbarIcon();
-            using (var iconStream = GetResourceStream(new Uri("pack://application:,,,/ShokoServer;component/db.ico"))
-                ?.Stream)
-            {
-                if (iconStream != null) Icon.Icon = new Icon(iconStream);
-            }
-            Icon.ToolTipText = "Shoko Server";
-            ContextMenu menu = new ContextMenu();
-            MenuItem webui = new MenuItem();
-            webui.Header = "Open WebUI";
-            webui.Click += (sender, args) =>
-            {
-                try
-                {
-                    string IP = GetLocalIPv4(NetworkInterfaceType.Ethernet);
-                    if (string.IsNullOrEmpty(IP))
-                        IP = "127.0.0.1";
-
-                    string url = $"http://{IP}:{ServerSettings.Instance.ServerPort}";
-                    OpenUrl(url);
-                }
-                catch (Exception ex)
-                {
-                    Logger.Error(ex, ex.ToString());
-                }
-            };
-            menu.Items.Add(webui);
-            webui = new MenuItem();
-            webui.Header = "Exit";
-            webui.Click += (sender, args) => Dispatcher.Invoke(Shutdown);
-            menu.Items.Add(webui);
-            Icon.ContextMenu = menu;
-            Icon.MenuActivation = PopupActivationMode.All;
-            Icon.Visibility = Visibility.Visible;
+    private static          TaskbarIcon? _icon;
+    private static readonly Logger       Logger = LogManager.GetCurrentClassLogger();
```

Comment:
Making variables align like this imo is pretty pointless and it also doesn't happen to occur a lot in the rest of the shoko codebase.

Reply:
please provide a complete .editorconfig for fomating issues, this is ReSharper default / detection of Shoko's Style.

Reply:
That will be in the works but in the mean time, you can change this.